jib-core/src/main/java/com/google/cloud/tools/jib/json/ListOfJsonTemplate.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2018 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.json;
+
+import java.util.List;
+
+public interface ListOfJsonTemplate extends JsonTemplate {
```

Comment:
a short javadoc for what this is / how to implement

Reply:
Eh, it's probably fine how it is. It's probably better not to worry about hypotheticals in the API until we actually encounter those cases.